insights/config/specs.py

```diff
@@ -313,7 +314,9 @@
     "rhsm.conf"                 : SimpleFileSpec("etc/rhsm/rhsm.conf"),
     "rhsm.log"                  : SimpleFileSpec("var/log/rhsm/rhsm.log", large_content=True),
     "root_crontab"              : First([CommandSpec("/usr/bin/crontab -l -u root"),
-                                    SimpleFileSpec("sos_commands/crontab")]),
+                                    SimpleFileSpec("sos_commands/crontab"),
+                                    SimpleFileSpec("sos_commands/cron/root_crontab"),
+                                    SimpleFileSpec("sos_commands/crontab/root_crontab")]),
```

Comment:
The modification to line-234, 317, 318 and 319 should not be there. Please ensure your branch is based on the latest master branch.